### ISSUE TITLE
Error: inputenc is not designed for xetex or luatex. ^^I\end{tcblisting}

### ISSUE THREAD

**Issue by Komari-Koshigaya** [T-F-S/tcolorbox#159]
Doesn't tcblisting support XeLaTex?

I'm trying to using tcblisting in XeLaTex compiler, but there are mistakes like "**Package inputenc Error: inputenc is not designed for xetex or luatex. ^^I\end{tcblisting}**"

related code
```tex
\documentclass{ctexart}
\usepackage[breakable]{tcolorbox}
\tcbuselibrary{listings}

\begin{document}
\begin{tcblisting}{title=example}
	This is a \LaTeX\ example:
	\begin{equation}
		\sum\limits_{i=1}^n i = \frac{n(n+1)}{2}.
	\end{equation}
\end{tcblisting}

\end{document}
```

Corresponding picture
![image](https://user-images.githubusercontent.com/39209375/145205401-6c0897b0-6446-4401-ba67-bf7a1d50237c.png)

**Comment by muzimuzhi** [comment]
Add command-line option `-8bit` to `xelatex`, see package manual of `minted`, sec 7 "FAQ and Troubleshooting",  "Tabs are being turned into the character sequence `^^I`".

**Comment by Komari-Koshigaya** [comment]
@muzimuzhi This solution works in command-line , but doesn't work for TeXstudio where I set **default compiler: `txs:///xelatex -8bit`**,

**Comment by muzimuzhi** [comment]
You should modify texstudio's command `txs:///xelatex`. By calling `txs:///xelatex -8bit`, `-8bit` is appended _after_ the tex file, like `xelatex synctex=1 -interaction=nonstopmode main.tex -8bit`, hence dropped.

**Comment by Komari-Koshigaya** [comment]
@muzimuzhi But I got this error(**^^I\end{tcblisting}**) when I originally set **default compiler `txs:///xelatex`**。
I got it, and solve the problem in texsudio by modifying command `xelatex -synctex=1 -interaction=nonstopmode -8bit %.tex`. 
But this command is too long, is there a better way?

**Comment by muzimuzhi** [comment]
> I got it, and solve the problem in texsudio by modifying command `xelatex -synctex=1 -interaction=nonstopmode -8bit %.tex`.
> But this command is too long, is there a better way?

No. Calling `xelatex <options> -8bit <tex file>` is unavoidable. If you'd even seen a shorter command, that's because you didn't unbox the abstraction.

This is `tcolorbox`'s issue tracker, and we're off-topic. Further Q&A should happen in general forums like [tex.stackexchange.com](https://tex.stackexchange.com/).

**Comment by Komari-Koshigaya** [comment]
Yes, you are right. At the beginning, I think the problem is about tcolorbox. 
Thanks for your suggestions.
